acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/keymanager/DeleteKeystoresAcceptanceTest.java

```diff
@@ -180,6 +189,66 @@ public void deletingReadOnlyKeyReturnError() throws URISyntaxException {
         .body("slashing_protection", is(emptySlashingData));
   }
 
+  @Test
+  public void deletingDisablesSigningForAllWeb3Signers()
+      throws URISyntaxException, JsonProcessingException {
+    final String firstPubkey = createBlsKey("eth2/bls_keystore.json", "somepassword");
+    final String secondPubKey = createBlsKey("eth2/bls_keystore_2.json", "otherpassword");
+    setupSignerWithKeyManagerApi(true);
+
+    final SignerConfiguration signer2Configuration =
+        new SignerConfigurationBuilder()
+            .withKeyStoreDirectory(testDirectory)
+            .withMode("eth2")
+            .withNetwork("minimal")
```

Comment:
nvm - found it :) https://github.com/ethereum/consensus-specs/tree/master/configs

Reply:
Would be interesting to know what impact setting that has on this particular test (compared to when it wasn't set before)

Reply:
For this test it's only needed because I wanted to sign a V2 block. Previously this test didn't have any need to sign blocks. I only affects tests that are signing and specifically require use of the forks. The import test also does has signing part to it but only signs an attestation that is unaffected by this change.

Reply:
makes sense, thanks for the explanation